web/static/js/contributors.js

```diff
@@ -1,5 +1,22 @@
-$(document).ready(function(){
-  jQuery.get('https://api.github.com/repos/codethesaurus/codethesaur.us/contributors?accept=application/vnd.github.v3+json&per_page=100&anon=1', function(data, status){
-    $("#contributors").text(data.length)
-  })
-})
\ No newline at end of file
+$(document).ready(function () {
+  let i = 1;
+  let length = 0;
+  const sleep = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
+  const timer = async () => {
+    await sleep(1000);
```

Comment:
Waiting before the request seems to me like wasting time can we move this down to before L17 where we call timer again? Maybe something like `setTimeout(timer, 1000)` in L17 ?
Also can we maybe get away with a shorter delay between calls 1s seems long to me but it might be necessary for the github api?

Reply:
~~Waiting before the request seems to me like wasting time can we move this down to before L17 where we call timer again? Maybe something like setTimeout(timer, 1000) in L17 ?~~
Done.

_Also can we maybe get away with a shorter delay between calls 1s seems long to me but it might be necessary for the github api?_
I cut it down to 500ms. I believe this will be sufficient for now.